frontend/components/siteList/siteListItem.jsx

```diff
@@ -41,6 +41,16 @@ const SiteListItem = ({ site, user }) => (
         </Link>
         {' '}
       </h4>
+      {
+        organization ? (
+          <h6>
+            <Link to={`/organizations/${organization.id}`} title="View organization settings">
```

Comment:
This is currently a link to a page that doesn't exist, maybe let's not make it a link for now and present any "org" settings differently for org managers